### PR TITLE
Committing changes for signInFrequency updates

### PR DESCRIPTION
**Pull request by rckyplln**

### CODE REVIEW

**Comment by Lauragra** [comment]
@rckyplln , we will need a changelog entry for the new properties/enum types. Also requesting that @FaithOmbongi review and approve. Thanks.

**Comment by rckyplln** [comment]
Hi all,

Apologies for the delay, here’s the changelog information: Pull request 5795065: Public preview reauth - Repos (visualstudio.com)<https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5795065?_a=files&path=%2FChangelog%2F5795065-Public-preview-reauth%2FMicrosoft.IdentityProtectionServices.json>
I had to make one last update today and put the isEnabled attribute back. The latest commit is there.

Let me know what else is needed. I apologize for all of the back and forth—this process is all new to me! 😅

/ricky

**Comment by rckyplln** [comment]
Here is the changelog, as requested: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5795065?_a=files&path=/Changelog/5795065-Public-preview-reauth/Microsoft.IdentityProtectionServices.json

**Comment by Lauragra** [comment]
@rckyplln , please follow the [Get started](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub) steps to set yourself up to be a contributor. Then, you can add the changelog entry that was generated directly to this file in your branch: https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/changelog/Microsoft.IdentityProtectionServices.json. When you're ready for us to review your PR, please set the **ready for content review** label. Thanks.

**Comment by rckyplln** [comment]
Hi Laura,

Thank you for your guidance here. I went through the getting started guide, and added myself to the various Groups and Security Groups. I added the labels as request below and also created an ADO work item: Product Backlog Item 1821923: [PROPOSED] Sign-in frequency API updates for re-authentication feature release - Boards (visualstudio.com)<https://identitydivision.visualstudio.com/Technical%20Content/_workitems/edit/1821923>.

Thank you again!
/ricky

**Comment by rckyplln** [comment]
#feedback-addressed

File: api-reference/beta/resources/signinfrequencysessioncontrol.md [hunk 1]
```diff
@@ -20,8 +20,10 @@ Session control to enforce sign-in frequency. Inherits from [Conditional Access
 | Property     | Type        | Description |
 |:-------------|:------------|:------------|
 |isEnabled     |Boolean      | Specifies whether the session control is enabled. |
-|type          |signinFrequencyType       | Possible values are: `days`, `hours`.|
+|type          |signinFrequencyType       | Possible values are: `days`, `hours`, or `null` if frequencyInterval is everyTime .|
 |value         |Int32        | The number of `days` or `hours`.|
+|authenticationType |signInFrequencyAuthenticationType  | Possible values are `primaryAndSecondaryAuthentication` or `secondaryAuthentication` .|
+|frequencyInterval  |signInFrequencyInterval  | Possible values are `timeBased` or `everyTime` .|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/signinfrequencysessioncontrol.md:26]: `unknownFutureValue` helps customers know these are evolvable enums and they can handle future changes in their code
```suggestion
|authenticationType |signInFrequencyAuthenticationType  | The possible values are `primaryAndSecondaryAuthentication`, `secondaryAuthentication`, `unknownFutureValue`.|
|frequencyInterval  |signInFrequencyInterval  | The possible values are `timeBased`, `everyTime`, `unknownFutureValue`.|
```

File: api-reference/beta/resources/signinfrequencysessioncontrol.md [hunk 2]
```diff
@@ -42,9 +44,10 @@ The following is a JSON representation of the resource.
 
 ```json
 {
-  "isEnabled": true,
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/signinfrequencysessioncontrol.md:45]: Please restore the **isEnabled** property, the type will be "Boolean" though - this property is inherited from the **conditionalAccessSessionControl** base type. Unless it's not returned in the response payload - then that's a different conversation.

File: changelog/Microsoft.IdentityProtectionServices.json [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+				"Target": "signInFrequencySessionControl"
+			  },
+			  {
+				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
+				"ApiChange": "Property",
+				"ChangedApiName": "frequencyInterval",
+				"ChangeType": "Addition",
+				"Description": "Added the [frequencyInterval](https://docs.microsoft.com/en-us/graph/api/resources/signInFrequencySessionControl?view=graph-rest-beta) resource",
+				"Target": "signInFrequencySessionControl"
+			  }
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityProtectionServices.json:36]: The descriptions were off. If they were autogenerated like this by Graph Studio, please share in ADO the link to the schema PR so I can file a bug report with the onboarding team.
```suggestion
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Enum type",
				"ChangedApiName": "signInFrequencyAuthenticationType",
				"ChangeType": "Addition",
				"Description": "Added the **signInFrequencyAuthenticationType** enumeration.",
				"Target": "signInFrequencyAuthenticationType"
			  },
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Enum type",
				"ChangedApiName": "signInFrequencyInterval",
				"ChangeType": "Addition",
				"Description": "Added the **signInFrequencyInterval** enumeration.",
				"Target": "signInFrequencyInterval"
			  },
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Property",
				"ChangedApiName": "authenticationType",
				"ChangeType": "Addition",
				"Description": "Added the **authenticationType** property to the [signInFrequencySessionControl](https://docs.microsoft.com/en-us/graph/api/resources/signInFrequencySessionControl?view=graph-rest-beta) resource.",
				"Target": "signInFrequencySessionControl"
			  },
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Property",
				"ChangedApiName": "frequencyInterval",
				"ChangeType": "Addition",
				"Description": "Added the **frequencyInterval** property to the [signInFrequencySessionControl](https://docs.microsoft.com/en-us/graph/api/resources/signInFrequencySessionControl?view=graph-rest-beta) resource.",
				"Target": "signInFrequencySessionControl"
			  }
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @rckyplln for your work in this PR. A few fixes and the PR will be good for approval.

**Comment by rckyplln** [comment]
Was getting a merge error-- appears to be fixed. Ready for review once more. Thanks for your help and patience on this!

**Review by FaithOmbongi — APPROVED** [review]

**Comment by rckyplln** [comment]
@FaithOmbongi , not sure what the above means-- is any action required from me?

**Comment by FaithOmbongi** [comment]
Hi @rckyplln, you'll need to resolve the merge conflict to get your PR merged when you're ready to merge it.

**Pull request merged**